src/electron.js

```diff
@@ -30,12 +29,8 @@ if (process.env.NODE_ENV === 'development') {
   }
 }
 
-// In production, we need to use `fixPath` to let Guppy use NPM.
-// For reasons unknown, the opposite is true in development; adding this breaks
-// everything.
 if (process.env.NODE_ENV !== 'development') {
   icon256Path = '../build/256x256.png';
-  fixPath();
```

Comment:
I tried to use fixPath in various locations in this file (outside of the if statement) to see if it was simply not working due to not having access to `NODE_ENV`, but no combination was working, so I've just removed it.